spring/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2016 LINE Corporation. All rights reserved.
+  ~ LINE Corporation PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+  -->
```

Comment:
I'm not sure what was in this file (@imasahiro FWIW I've settled on not doing amend and only rare rebases after releasing a PR to improve the github experience :) ), but if we can provide best practices for free, may as well. Though of course for logging, this will be fairly limited - e.g., logging to a file is not needed in container environments, so if it was that sort of configuration here, then agree it's not needed.